wwwroot/fileupload.html

```diff
@@ -122,6 +184,139 @@ <h1 class="mt-4">File Upload</h1>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/simple-datatables@7.1.2/dist/umd/simple-datatables.min.js" crossorigin="anonymous"></script>
         <script src="js/site.js"></script>
-        <script>getFiles();</script>
+        <script>
+        function getFiles() {
```

Comment:
Since this is self invoking, I recommend using a IIFE

https://developer.mozilla.org/en-US/docs/Glossary/IIFE

(()=>{
   //code goes here
})()